tests/wpunit/Admin/Fields/FieldTest.php

```diff
@@ -213,4 +245,35 @@ public function it_should_handle_empty_label( $resource ) {
 
 		$this->assertEmpty( $field->get_label(), 'Label should be empty by default' );
 	}
+
+	/**
+	 * @test
+	 */
+	public function it_should_render_correct_html_for_oauth_resource_with_license_key() {
+		$this->set_fn_return( 'wp_create_nonce', '535281edcd' );
+		$this->set_fn_return( 'wp_generate_password', '535281edcd535281ed' );
+		$slug = 'service-oauth-with-license-key-field-1';
+		Register::service(
+			$slug,
+			'Service OAuth With License Key Field 1',
+			'1.0.0',
+			$this->get_base() . '/service-oauth-with-license-key-field-1.php',
+			Uplink::class,
+			Uplink::class,
+			Resource::OAUTH_REQUIRES_LICENSE_KEY
+		);
+		add_filter( 'stellarwp/uplink/test/auth/can_auth', '__return_true' );
```

Comment:
This is getting cleaned up after the test ?

Reply:
Yes, like all filters they are being reset to the original between tests.